### PR TITLE
feat: local config fallback for invalid edge config

### PR DESCRIPTION
**Pull request by jeromeandrewong**
closes #ISSUE_NUMBER

## Context

- add fallback to use local json config if dev doesnt' have the correct edge config env token, so development workflow isn't blocked

## How to Test

- app should work locally with /without or correct/incorrect `EDGE_CONFIG` variable in `.env` file
## Checklist

- [ ] I have read the [CONTRIBUTING](CONTRIBUTING.md) document
- [ ] I have tested the changes
- [ ] _(where applicable)_ I have added tests to cover my changes
- [ ] _(where applicable)_ I have updated the documentation accordingly

### CODE REVIEW

File: src/common/providers/EdgeConfig/EdgeConfigProvider.tsx [hunk 1]
```diff
@@ -1,33 +1,44 @@
...
+
 export async function EdgeConfigProvider({
   children,
 }: {
   children: React.ReactNode;
 }) {
-  // * FUTURE NOTE *
-  // if edge requests count nears the threshold,
-  // we should consider caching the edge config & revalidate every 24h
-  const edgeConfigRaw = await getAll();
```
Comment (davidlhw) [inline src/common/providers/EdgeConfig/EdgeConfigProvider.tsx:14]: could we leave this comment in? :D

it'll be a useful reference for the future if/when we need to do performance optimization for it

**Review by davidlhw — APPROVED** [review]
CLGTM @ cf6cb7c3b8fdcd187b23f65ca037e81fef682845

just a minor comment, good to merge

File: src/common/providers/EdgeConfig/EdgeConfigProvider.tsx (hunk 1 above)
Comment (jeromeandrewong) [reply src/common/providers/EdgeConfig/EdgeConfigProvider.tsx:14]: added back! [8e00293](https://github.com/AfterClass-io/afterclass.io-v2/pull/345/commits/8e00293559b7beb47f5ee3f3d793d8a626b966b0)

**Review by davidlhw — APPROVED** [review]
CLGTM @ 8e00293559b7beb47f5ee3f3d793d8a626b966b0

**Pull request merged**